### PR TITLE
minor stuff

### PR DESCRIPTION
**Pull request by rigid**
- use short notation for conditionals
- remove unneeded commented sections

### CODE REVIEW

File: sci-libs/opencascade/opencascade-7.3.0.ebuild [hunk 1]
```diff
@@ -80,7 +80,6 @@ src_prepare() {
 }
 
 src_configure() {
-#		-DBUILD_USE_PCH=$(usex pch)
```
Comment (waebbl) [inline sci-libs/opencascade/opencascade-7.3.0.ebuild]: Please leave this line. It's attached to the TODO on top of the file. I want to test whether this works and eventually build it into the ebuild. It's easier to have the necessary line around than to rewrite it ;)

File: sci-libs/opencascade/opencascade-7.3.0.ebuild [hunk 2]
```diff
@@ -98,25 +97,11 @@ src_configure() {
 		-DUSE_GLES2=$(usex gles2)
 		-DUSE_TBB=$(usex tbb)
 		-DUSE_VTK=$(usex vtk)
+		-DINSTALL_SAMPLES=$(usex examples)
```
Comment (waebbl) [inline sci-libs/opencascade/opencascade-7.3.0.ebuild]: Would you care to move to up a little, so the args are sorted alphabetically? I.e. put it above -DINSTALL_TEST_CASES.

File: sci-libs/opencascade/opencascade-7.3.0.ebuild [hunk 3]
```diff
@@ -98,25 +97,11 @@ src_configure() {
...
-			-DINSTALL_SAMPLES="yes"
-		)
-		if use qt5; then
-			mycmakeargs+=(
-				-DBUILD_SAMPLES_QT="yes"
-			)
-		fi
-	fi
-
+	use example && mycmakeargs += ( -DBUILD_SAMPLES_QT=$(usex qt5) )
```
Comment (waebbl) [inline sci-libs/opencascade/opencascade-7.3.0.ebuild]: It should read use example**s**

File: sci-libs/opencascade/opencascade-7.3.0.ebuild (hunk 1 above)
Comment (rigid) [reply sci-libs/opencascade/opencascade-7.3.0.ebuild]: ah, ok. Maybe the comment could be placed next to here?

**Pull request merged**